src/modules/previewpane/PdfPreviewHandler/loc/nl/src/modules/previewpane/PdfPreviewHandler/Properties/Resources.resx.lcl

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file needs to be removed.
Localization is done with an internal process.

Reply:
It is gone.